src/components/application/application.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import {Container, H1, Image, Text, ApplicationButton, RoundButton} from "./style";
+import Play from "../icons/play/play"
+
+interface IApplicationProp {
+  image: string;
+  h1: string;
+  text: string;
+  onClick: () => void;
+}
+
+const Application = ({ image, h1, text, onClick}: IApplicationProp) => {
+
+  return (
+    <Container>
+      <H1>Проект “Английский центр”</H1>
```

Comment:
If i want change project name, how i will do it ? Pls use props for titles where it need